addons/l10n_nl/migrations/3.3/post-migrate_update_taxes.py

```diff
@@ -0,0 +1,82 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, SUPERUSER_ID
+
+
+def migrate(cr, version):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    goods_taxes = env['account.tax'].search([('name', '=', 'Verkopen export binnen EU (producten)')])
+    services_taxes = env['account.tax'].search([('name', '=', 'Verkopen export binnen EU (diensten)')])
+
+    old_plus_tax_tag_id = env['account.account.tag'].search([('name', '=', '+3bl (omzet)')], limit=1).id
```

Comment:
```suggestion
    old_plus_tax_tag_id = env['account.account.tag'].search([('name', '=', '+3bl (omzet)')], limit=1).id
```
There is a dedicated method for that :wink: : `_get_tax_tags()`

Reply:
I'll have to use ```_get_tax_tags_domain()``` instead be cause I need a way to differentiate the sign